plugins/koterm.koplugin/main.lua

```diff
@@ -0,0 +1,714 @@
+--[[
+This plugin provides a terminal emulator (VT52 (+some ANSI))
+]]
+
+local Device = require("device")
+
+-- grantpt and friends are necessary (introduced on Android in API 21).
+-- So sorry for the Tolinos with (Android 4.4.x)
+if Device:isAndroid() then
+    local A, android = pcall(require, "android")  -- luacheck: ignore
+    local api = android.app.activity.sdkVersion
+    if api < 21 then
+        return { disabled = true }
+    end
+end
+
+local Dispatcher = require("dispatcher")
+local Font = require("ui/font")
+local InfoMessage = require("ui/widget/infomessage")
+local InputDialog = require("ui/widget/inputdialog")
+local MultiConfirmBox = require("ui/widget/multiconfirmbox")
+local ScrollTextWidget = require("ui/widget/scrolltextwidget")
+local SpinWidget = require("ui/widget/spinwidget")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local TermInputText = require("terminputtext")
+local TextWidget = require("ui/widget/textwidget")
+local T = require("ffi/util").template
+local bit = require("bit")
+local logger = require("logger")
+local _ = require("gettext")
```

Comment:
Not a big deal, but let's keep the coding style as in the style guide (functions under _) lest people copy this impressive piece of art. 
```suggestion
local bit = require("bit")
local logger = require("logger")
local _ = require("gettext")
local T = require("ffi/util").template
```